x-pack/legacy/plugins/maps/public/actions/map_actions.js

```diff
@@ -753,9 +753,9 @@ export function clearMissingStyleProperties(layerId) {
       return;
     }
 
-    const ordinalFields = await targetLayer.getOrdinalFields();
+    const ordinalAndCategoricalFields = await targetLayer.getOrdinalAndCategoricalFields();
```

Comment:
How about just using `layer.getFields`? Really this check should be against all fields, not just ordinal and categorical since labels is using `getFields`.